packages/lit-dev-tools-cjs/src/playground-plugin/blocking-renderer.ts

```diff
@@ -70,6 +121,11 @@ export class BlockingRenderer {
       htmlBuffer: this.sharedHtml,
       notify: this.sharedNotify,
     });
+    try {
+      fs.mkdirSync(cachedHighlightsDir);
+    } catch {
+      // Directory already exists.
+    }
```

Comment:
Nit: Just in case not to swallow other errors?
```suggestion
    try {
      fs.mkdirSync(cachedHighlightsDir);
    } catch (error) {
      if (!(error as {code: string}).code !== 'EEXIST') {
        throw error;
      }
    }
```

Reply:
Alternatively...
```suggestion
    try {
      fs.mkdirSync(cachedHighlightsDir);
    } catch (error) {
      if ((error as {code: string}).code === 'EEXIST') {
        // Directory already exists.
      } else {
        throw error;
      }
    }
```

Reply:
I like the latter that throws! Thank you!